### ISSUE TITLE
rectlabel for mac 10.11

### ISSUE THREAD

**Issue by bluefox994** [ryouchinsa/Rectlabel-support#20]
The app is only available for mac version 10.12 or above I have a 10.11 can I still use this?

**Comment by ryouchinsa** [comment]
Thanks for writing the issue.
Could you use RectLabel on your mac?

**Comment by bluefox994** [comment]
I can. Thank you; I upgraded the OS does this support video annotation as well?
Thanks!
Maddie

On Sep 13, 2017, at 3:25 PM, Ryo Kawamura <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks for writing the issue.
Could you use RectLabel on your mac?

**Comment by ryouchinsa** [comment]
We have not supported the video file input yet.
To support the video file input, could you tell us some information about the video annotation?

**Comment by bluefox994** [comment]
Yes I'm going to be annotating surveillance footage of a Taco Bell so the background should be pretty static. Let me know and thanks so much!

On Sep 13, 2017, at 9:21 PM, Ryo Kawamura <notifications@github.com<mailto:notifications@github.com>> wrote:


We have not supported the video file input yet.
To support the video file input, could you tell us some information about the video annotation?

**Comment by ryouchinsa** [comment]
We checked some competitive products.
http://carlvondrick.com/vatic/
https://www.youtube.com/watch?v=pB26ZMSp-XI
https://github.com/CatalystCode/VoTT

Do you think these annotating flows would match with your need?

When we would implement the video annotations, we have to know the detailed use case.

At this moment, if you could convert the video file into an image sequence in a folder, you can open the folder with RectLabel.

**Comment by bluefox994** [comment]
I looked into Vatic but your tool just looked awesome so I was hoping. VoTT looks promising. I have experience in data science and I know this computer vision problem I'm trying to tackle is going to be tough but I'm hoping to figure it out. I'm trying to understand suspicious behavior in videos. I'm starting out with Taco Bell data to understand loss prevention, by checking that what the cashiers give out (number of items) match point of sale receipt. So I will have to be counting these items and food is small so the bounding boxes will be small. Then, I'm thinking about implementing this is with fcrnn and pytorch. I then am thinking for tracking to use iou comparisons. This is a side project I'm working on and if you have any advice suggestions I would be extremely grateful. Thank you!
Maddie

On Sep 13, 2017, at 10:26 PM, Ryo Kawamura <notifications@github.com<mailto:notifications@github.com>> wrote:


We checked some competitive products.
http://carlvondrick.com/vatic/
https://www.youtube.com/watch?v=pB26ZMSp-XI
https://github.com/CatalystCode/VoTT

Do you think these annotating flows would match with your need?

When we would implement the video annotations, we have to know the detailed use case.

At this moment, if you could convert the video file into an image sequence in a folder, you can open the folder with RectLabel.

**Comment by bluefox994** [comment]
Do you think this is possible and if so within what time frame? If not, I'll go ahead and try out your other suggestions! Thanks! 
Maddie

**Comment by ryouchinsa** [comment]
This blog post is very useful.

How to train your own Object Detector with TensorFlow’s Object Detector API
https://medium.com/towards-data-science/how-to-train-your-own-object-detector-with-tensorflows-object-detector-api-bec72ecfe1d9

You have to go through these processes.
- Prepare a AWS or Google Cloud server with GPU and install tensorflow.
- Labelling the video image sequence in the PASCAL VOC format.
- Train the data with tensorflow.
- Apply the trained model to the new video image sequence or video input.

Hopefully it would take a few months.

**Comment by bluefox994** [comment]
Hi Ryo, what would take a few months? Do you have time for a call today? I am hoping to use pytorch. What do you think about this?
Let me know and thanks!
Maddie

On Sep 14, 2017, at 10:53 AM, Ryo Kawamura <notifications@github.com<mailto:notifications@github.com>> wrote:


This blog post is very useful.

How to train your own Object Detector with TensorFlow’s Object Detector API
https://medium.com/towards-data-science/how-to-train-your-own-object-detector-with-tensorflows-object-detector-api-bec72ecfe1d9

You have to go through these processes.

  *   Prepare a AWS or Google Cloud server with GPU and install tensorflow.
  *   Labelling the video image sequence in the PASCAL VOC format.
  *   Train the data with tensorflow.
  *   Apply the trained model to the new video image sequence or video input.

Hopefully it would take a few months.

**Comment by ryouchinsa** [comment]
You told us that this was a side project so that we thought you would not use your full time onto  this project.

If you could work full time every day:

> - Prepare a AWS or Google Cloud server with GPU and install tensorflow.

It would take a few days.

> - Labelling the video image sequence in the PASCAL VOC format.

It would take a few days for the first 200-300 images

> - Train the data with tensorflow.

You might need some trial and errors to increase the accuracy.
It might take a week.

> - Apply the trained model to the new video image sequence or video input.

If you could use some useful sample code to process video input, it would take a few days.
Otherwise you have to code to process the video input with your model, it might take a week.

If you have some experience on deep learning and object detection, it might be soon.

**Comment by ryouchinsa** [comment]
Now you can convert video to image frames.
https://rectlabel.com/help#video_to_frames

![video_frames](https://user-images.githubusercontent.com/1954306/51837558-12688280-2347-11e9-8846-4885597b7920.jpg)
